torch/_inductor/codegen/common.py

```diff
@@ -511,6 +511,185 @@ def remainder(a, b):
+            impl = impls.get(target)
+            if impl is not None:
+                nof_args = 2 if "{y}" in impl else 1
+                # extend the following dictionary with factory
+                # functions for a specific number of arguments as
+                # needed:
+                factory = {1: pointwise_factory_1, 2: pointwise_factory_2}[nof_args]
+                setattr(cls, funcname, staticmethod(factory(impl)))
+
+
+pointwise_overrides_data: Dict[str, Dict[str, str]] = dict(
+    airy_ai=dict(
+        cpp="airy_ai_forward({x})",
+        # triton=...,  # libdevice/triton do not provide Airy Ai function
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_airy_ai",
+    ),
+    bessel_j0=dict(
+        cpp="bessel_j0_forward({x})",
+        triton="tl.math.j0({x})",
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_bessel_j0",
+    ),
+    bessel_j1=dict(
+        cpp="bessel_j1_forward({x})",
+        triton="tl.math.j1({x})",
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_bessel_j1",
+    ),
+    bessel_y0=dict(
+        cpp="bessel_y0_forward({x})",
+        triton="tl.math.y0({x})",
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_bessel_y0",
+    ),
+    bessel_y1=dict(
+        cpp="bessel_y1_forward({x})",
+        triton="tl.math.y1({x})",
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_bessel_y1",
+    ),
+    digamma=dict(
+        cpp="calc_digamma({x})",
+        # triton=...,  # libdevice/triton do not provide digamma function
+        cppvec="{x}.digamma()",
+        aten="digamma",
+    ),
+    # no cpp nor triton implementation for entr, it is defined as decomposition
+    # erf, erfc
+    erfcx=dict(
+        cpp="calc_erfcx({x})",
+        triton="tl.math.erfcx({x})",
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_erfcx",
+    ),
+    # erfinv, exp2, expit, gammainc, gammaincc, gammaln
+    i0=dict(
+        cpp="calc_i0({x})",
+        triton="tl.math.cyl_bessel_i0({x})",
+        cppvec="{x}.i0()",
+        aten="i0",
+    ),
+    i0e=dict(
+        cpp="calc_i0e({x})",
+        # triton=...,  # libdevice/triton do not provide i0e function
+        cppvec="{x}.i0e()",
+        aten="special_i0e",
+    ),
+    i1=dict(
+        cpp="calc_i1({x})",
+        triton="tl.math.cyl_bessel_i1({x})",
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_i1",
+    ),
+    i1e=dict(
+        cpp="calc_i1e({x})",
+        # triton=...,  # libdevice/triton do not provide i1e function
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_i1e",
+    ),
+    log_ndtr=dict(
+        cpp="calc_log_ndtr({x})",
+        # triton=...,  # libdevice/triton do not provide log_ndtr function
+        # cppvec=...,  # not impl in aten/.../vec
+        aten="special_log_ndtr",
+    ),
+    # log_softmax
+    # logit
+    # logsumexp
```

Comment:
`log_softmax` and `logsumexp` aren't pointwise operators.

Reply:
Right, the references to these are now removed.